app/lib/ui/views/account/account_editing_screen.dart

```diff
@@ -69,6 +69,38 @@ class _AccountEditingScreenState extends State<AccountEditingScreen> {
                               color: NavyBlue,
                               fontWeight: FontWeight.bold),
                         ),
+                        Padding(
+                          padding: EdgeInsets.only(
+                            top: 20.0,
+                          ),
+                        ),
+                        CircleAvatar(
+                          radius: 53,
+                          backgroundColor: NavyBlue,
+                          child: model.selectedImage != null
+                              ? ClipRRect(
+                                  borderRadius: BorderRadius.circular(50),
+                                  child: Image.file(
+                                    model.selectedImage,
+                                    width: 100,
+                                    height: 100,
+                                    fit: BoxFit.cover,
+                                    scale: 1.0,
+                                  ),
+                                )
+                              : Container(
+                                  decoration: BoxDecoration(
+                                    color: Colors.grey[300],
+                                    borderRadius: BorderRadius.circular(50),
+                                  ),
+                                  width: 100,
+                                  height: 100,
+                                  child: Icon(
+                                    Icons.camera_alt,
```

Comment:
- [ ] When I see the camera Icon, it makes me wanna tap it. You should add an `onPressed` here to pick the image too, like the button below

Reply:
Approach to solve this problem: I'll use gesture detector here.